java/test/jmri/jmrit/display/layoutEditor/load/LayoutEditorTest.xml

```diff
@@ -219,50 +219,51 @@
         <rotation>0</rotation>
       </icon>
     </positionablelabel>
-    <layoutturnout ident="TO4" disabled="yes" turnoutname="IT1" blockname="North" type="1" connectaname="T1" connectbname="T12" connectcname="T10" continuing="2" disableWhenOccupied="no" xcen="202.0" ycen="40.0" xa="184.0" ya="40.0" xb="220.0" yb="40.0" xc="220.0" yc="50.0" xd="184.0" yd="40.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" />
-    <layoutturnout ident="TO8" hidden="yes" turnoutname="IT2" blockname="North" type="2" connectaname="T15" connectbname="T12" connectcname="T11" continuing="2" disabled="no" disableWhenOccupied="no" xcen="320.0" ycen="40.0" xa="360.0" ya="40.0" xb="280.0" yb="40.0" xc="300.0" yc="50.0" xd="360.0" yd="40.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" />
-    <layoutturnout ident="TO1" disableWhenOccupied="yes" turnoutname="IT7" blockname="Yard" type="5" connectaname="T1" connectbname="T4" connectcname="T2" connectdname="T6" continuing="2" disabled="no" xcen="210.0" ycen="250.0" xa="180.0" ya="240.0" xb="219.99999" yb="240.0" xc="240.0" yc="260.0" xd="200.00001" yd="260.0" ver="1" class="jmri.jmrit.display.layoutEditor.configurexml.LayoutTurnoutXml" />
```

Comment:
If you look at the Turnouts, you can see that not only did the order of the turnouts in the file change, but so did the order of the attributes.

There isn't anything wrong with changing the order of things in the file, but we need to verify both orders work.